composer.json

```
@@ -24,14 +24,17 @@
   "extra" : {
     "tao-extension-name" : "taoLtiConsumer"
   },
+  "require-dev": {
+    "php-mock/php-mock": ">=2.3.1"
```

Let's use proper semver dependencies where we can.
```suggestion
    "php-mock/php-mock": "^2.3"
```